packages/shared/src/components/post/PostComments.tsx

```diff
@@ -58,7 +59,7 @@ export function PostComments({
         requestMethod(
           graphqlUrl,
           POST_COMMENTS_QUERY,
-          { postId: id, [initialDataKey]: comments, first: 500 },
+          { postId: id, [initialDataKey]: comments, first: 50 },
```

Comment:
Not against it, but why did we change this here?

Reply:
Our post comments are not yet paginated, this would not show comments beyond 50.